login.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+	<!-- CSSs -->
+	<link rel="stylesheet" href="css/bootstrap.css">
+	<link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/all.css">
+  <link rel="stylesheet" href="style1.css"> 
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
+
+    
+    <title>Login</title>
+</head>
+
+<body data-gr-c-s-loaded="true">
+    <!-- Navbar -->
+    <nav class="navbar navbar-expand-lg navbar-light">
+        <div class="container">
+            <a class="navbar-brand" href="#"><img src="img/logo.png"></a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNav">
+              <ul class="navbar-nav">
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Categories</a>
+                </li>
+                <li class="nav-item">
+                    <div class="btn-post-job">
+                        <a class="nav-link" href="#">Post job</a>
+                    </div>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Contact</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">About us</a>
+                </li>
+              </ul>
+              <ul class="navbar-nav navbar-right">
+                  <li class="nav-item" style="margin: 0px !important;">
+                  <a class="nav-link" href="#">Login</a>
+                </li>
+                <li class="nav-item" style="margin: 0px !important;">
+                  <a class="nav-link" href="#">|</a>
+                </li>
+                <li class="nav-item" style="margin: 0px !important;">
+                  <a class="nav-link" href="#">Register</a>
+                </li>
+              </ul>
+            </div>
+        </div>
+      </nav>
+
+      <!-- main -->
+      <main>
+          <!-- logo icon -->
+          <div class="logo-icons">
+              <h1>Login</h1>
+              <h2>to find your future job</h2>
+          </div>
+
+          <!-- form -->
+        <div class="">
+          <div class="container">
+            <form>
+              <div class="form-group">
+                <label for="formGroupExampleInput">Email</label>
+                <input type="text" class="form-control" id="formGroupExampleInput" placeholder="Enter your email">

```

Comment:
👍  Gunakan atribute `for` dan `id`dengan jelas pada label dan input, serta tambahkan atribute `name` pada input. Koreksi :

```
<label for="email">Email</label>
<input type="text" class="form-control" id="email" placeholder="Enter your email" name='email'>
```